pennylane_lightning/src/bindings/Bindings.cpp

```diff
@@ -12,658 +12,253 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <cassert>
```

Comment:
Is this required for `static_assert` ? I think it is not needed as part of the standard.

Reply:
Yeap, let's remove this line